uvicorn/config.py

```diff
@@ -177,3 +172,8 @@ def load(self):
             self.ssl = None
 
         self.loaded = True
+
+    def setup_event_loop(self):
+        if isinstance(self.loop, str):
```

Comment:
It should always be a string now, so I think we can drop this branching.

Reply:
Removed. Some tests were still passing the loop directly.